Godeps/_workspace/src/github.com/openshift/source-to-image/pkg/api/types.go

```diff
@@ -91,6 +96,11 @@ type Config struct {
 	// Specify a relative directory inside the application repository that should
 	// be used as a root directory for the application.
 	ContextDir string
+
+	// AllowedUIDs is a list of user ranges of users allowed to run the builder image.
+	// If a range is specified and the builder image uses a non-numeric user or a user
+	// that is outside the specified range, then the build fails.
+	AllowedUIDs user.RangeList
```

Comment:
Yes, it can have a list of ranges like `10-20,30,50,1024-`. Ideally we'd be able to ask SCC what UID ranges are allowed for that user/sa and pass that on to the STI builder. But the way it is right now, I can only ask whether a particular user is allowed. @pweil - any thoguths?